scenedetect/scene_manager.py

```diff
@@ -489,11 +489,16 @@ def save_images(scene_list: List[Tuple[FrameTimecode, FrameTimecode]],
             frame_im = video.read()
             if frame_im is not None:
                 # TODO: Allow NUM to be a valid suffix in addition to NUMBER.
-                file_path = '%s.%s' % (filename_template.safe_substitute(
-                    VIDEO_NAME=video.name,
-                    SCENE_NUMBER=scene_num_format % (i + 1),
-                    IMAGE_NUMBER=image_num_format % (j + 1),
-                    FRAME_NUMBER=image_timecode.get_frames()), image_extension)
+                file_path = '%s.%s' % (
+                    filename_template.safe_substitute(
+                        VIDEO_NAME=video.name,
+                        SCENE_NUMBER=scene_num_format % (i + 1),
+                        IMAGE_NUMBER=image_num_format % (j + 1),
+                        FRAME_NUMBER=image_timecode.get_frames(),
+                        TIMESTAMP_MS=int(image_timecode.get_seconds() * 1000),
+                        TIMECODE=image_timecode.get_timecode()),
```

Comment:
Timecodes need to be formatted specially as not all operating systems support `:` in paths.  Please replace them with `;` [as is done when splitting videos](https://github.com/Breakthrough/PySceneDetect/blob/main/scenedetect/video_splitter.py#L140-L141):

```python
TIMECODE=image_timecode.get_timecode().replace(":", ";")
```

This is most likely why `test_save_images` is failing on the Windows build currently.